src/controllers/movie.js

```diff
@@ -0,0 +1,126 @@
+const { Prisma } = require("@prisma/client");
+const { request } = require("express");
+const { screening } = require("../utils/prisma");
+const prisma = require("../utils/prisma");
+const {
+  buildRuntimeClause,
+  errorMessages,
+  buildCreateMovieData,
+  checkFields,
+  updateMovieScreenings,
+} = require("./utils");
+
+const getAllMovies = async (req, res) => {
+  console.log("Queries: ", req.query);
+
+  const movies = await prisma.movie.findMany({
+    where: {
+      runtimeMins: buildRuntimeClause(req.query),
+      screenings: {
+        some: {
+          startsAt: {
+            gte: new Date(),
+          },
+        },
+      },
+    },
+    include: { screenings: true },
+  });
+
+  res.status(200).json({ movies });
+};
+
+const createMovie = async (req, res) => {
+  const { title, runtimeMins } = req.body;
+  if (checkFields([title, runtimeMins])) {
+    return res.status(400).json({ error: errorMessages.missingField });
+  }
+
+  // const found = await prisma.movie.findUnique({
+  //   where: {
+  //     title,
+  //   },
+  // });
+  // if (found) {
+  //   return res.status(409).json({ error: errorMessages.movieExists });
+  // }
```

Comment:
Commented code is considered "dead code" and should be removed before submitting a PR